site/es/blog/new-in-devtools-99/index.md

```diff
@@ -5,190 +5,222 @@ authors:
   - jecelynyeen
 date: 2022-02-21
 updated: 2022-02-21
-description: 'Throttling WebSocket requests, new Reporting API pane, console styling and more.'
+description: 'Regulación de peticiones WebSocket, nuevo panel de la API de Reportes, estilos en consola y más.'
 hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/mnEE3FiT5cpN2cclEcZ2.jpg'
 alt: ''
 tags:
   - new-in-devtools
   - devtools
   - chrome-99
-draft: true
 ---
 
-_Gracias [Carlos Caballero](https://carloscaballero.io) por la traducción y [Miguel Ángel](https://midu.dev) por la revisión._
+_Gracias [Miguel Ángel](https://midu.dev) por la traducción y [Carlos Caballero](https://carloscaballero.io) por la revisión._
 
 {% include 'partials/devtools/es/banner.md' %}
 
-
-<!-- start: translation instructions -->
-<!-- 1. Remove the "draft: true" tag above when submitting PR -->
-<!-- 2. Provide translations under each of the English commented original content -->
-<!-- 3. Translate the "description" tag above -->
-<!-- 4. Translate all the <img> alt text -->
-<!-- 5. Update the whats-new.md file -->
-<!-- end: translation instructions -->
-
 <!-- ## Throttling WebSocket requests {: #websocket } -->
-
+## Regulación de las peticiones WebSocket {: #websocket }
```

Comment:
Perhaps this concept is better understood using the translation "Limitación"? What do you think?